AlgebraInLean/World01/Sheet4.lean

```diff
@@ -0,0 +1,154 @@
+import AlgebraInLean.World01.Sheet3
+open Defs
+
+/- Now that we know how to define a group, and some basic properties of groups,
+we can find important examples of groups-/
+
+/- The group you are likely most familiar with is the integers, with addition
+as the group operation.-/
+instance : Defs.Group ℤ where
```

Comment:
To avoid typing `Defs.*` for everything, we should wrap everything in this file to be inside the `Defs` namespace. And while we're at it, we should rename this namespace to something more generic/global. Something like `AlgebraInLean`?